repairnator/src/main/java/fr/inria/spirals/repairnator/helpers/GitHelper.java

```diff
@@ -0,0 +1,190 @@
+package fr.inria.spirals.repairnator.helpers;
+
```

Comment:
I'm not fond of the package name: we already have some helpers for maven in repairnator/process/maven: we should follow the same pattern and put it in repairnator/process/git